iridauploader/api/api_calls.py

```diff
@@ -20,6 +20,17 @@
 from . import exceptions
 
 
+MODE_DEFAULT = "default"
+MODE_ASSEMBLIES = "assemblies"
+MODE_FAST5 = "fast5"
+
+UPLOAD_MODES = [
```

Comment:
Add a comment in here above these stating what they're for please.

Reply:
Added.